src/style.css

```
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  border-radius: 0.5px;
```

```suggestion
  border-radius: 0.5px;
```
do you really need this border-radius ? and if it needed use integers